packages/blockchain-api/package.json

```diff
@@ -17,7 +17,7 @@
     "deploy": "./deploy.sh"
   },
   "dependencies": {
-    "@celo/contractkit": "0.3.6",
+    "@celo/contractkit": "0.4.6-dev",
```

Comment:
bc-api can't use a -dev version because GAE installs the deps during deployment

Reply:
Thanks, good reminder!